README.md

```diff
@@ -31,3 +31,50 @@ explained in the config file.
 
 * How to run:
 `node scripts/fetchODCDataFiles.js https://data.ontario.ca/dataset/covid-19-assessment-centre-locations ODC`
+
+### Diff/merge 2 or 3 datasets
+
+* Script: scripts/test_daff.js
+
+* Goal: This script runs a diffing algorithm and produces html that shows a visulization of the diff and optionally an
+updated local data set that is a result of mergeing in the diff results into the local file. The visuliztion can be seen
+by running the produced html file in a browser. This script takes three arguments maximum but needs at least two arugments
+minimum to run successfully. Possible arguments are 1, 2, and 3. Corresponding to temporarily hard-coded dataset files. In
+future implementation arguments will be user selected files.
+
+* Arguments:
+  * first argument: URL of local data
+  * second argument: URL of remote data
+  * third argument: URL of ancestor data
+
+* How to run:
+`node scripts/test_daff.js 3 2 1`
+
+### Test diff 2
+
+* Script: scripts/test_diff2.js
```

Comment:
For test_diff2.js and test_diff3.js, they will accept arguments in file locations. For example:
```
node daff/test_diff2.js daff/data/weCount/ODC1.csv daff/data/weCount/WeCount1.csv daff/data/weCount/diff2_ODC1_WeCount1.html
```
So that people who run this script have control over input and output files rather than changing these values in the script.